lib/videoroom/meeting_supervisor.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Videoroom.MeetingSupervisor do
+  @moduledoc """
+  Supervisor for `Videoroom.Meeting`s,
+  providing functions for interacting with them.
+  """
+  use DynamicSupervisor
+
+  alias Videoroom.Meeting
+
+  @spec start_link(any()) :: Supervisor.on_start()
+  def start_link(_init_arg) do
+    DynamicSupervisor.start_link(__MODULE__, [], name: __MODULE__)
+  end
+
+  @impl true
+  def init(init_arg) do
+    DynamicSupervisor.init(init_arg)
+  end
+
+  @spec add_peer(binary()) :: {:ok, binary()} | {:error, binary()}
+  def add_peer(room_name) do
+    # `max_peers` = nil creates a room without limit
+    args = [name: room_name, max_peers: nil]
```

Comment:
Is there a plan to add feature that allows only specified number of peers in Videoroom meeting. If not, passing this `max_peers` that is always `nil` (which is quite gimmicky looking at `Metting.init/1`) makes things only harder IMO.